crates/iota-data-ingestion/src/workers/kv_store.rs

```diff
@@ -129,7 +123,13 @@ impl KVStoreWorker {
                     chunk.to_vec(),
                 )])))
                 .send()
-                .await?;
+                .await
+                .inspect_err(|sdk_err| {
+                    tracing::error!(
```

Comment:
Nit: you could import tracing

Reply:
fixed: https://github.com/iotaledger/iota/pull/6401/commits/0ba217427d37ed390deaaf10f6c50c8081ef7ce5